app/routes/events/components/RegisteredSummary.js

```diff
@@ -51,7 +51,7 @@ const RegistrationList = ({
     className={styles.registrationList}
     onClick={onClick}
   >
-    <a href="">
```

Comment:
why is this an `a` if there's no href or click handlers?  should probably just remove the tag altogether, and rather style `Tooltip`